bin/servicerouter.py

```diff
@@ -83,6 +83,10 @@
     An HTTP frontend that binds to port *:80 by default and gathers
     all virtual hosts as defined by the HAPROXY_{n}_VHOST variable.
 
+  HAPROXY_HTTP_FRONTEND_APPID_HEAD
+    An HTTP frontend that binds to port *:81 by default and gathers
+    all apps in http mode.
```

Comment:
Could you add documentation on how an app should use this. Something like:

To use this frontend to forward to your app, configure the app with "HAPROXY_0_MODE=http" then you can access it via  a call to the <servicerouter_host>:81 with the header "X-Marathon-App-Id" set to the Marathon AppId. Note multiple ports being exposed by the same marathon app are not supported


What happens when the same App exposes multiple ports as http? Can we explicitly emit a warning when this happens (and ignore the app entirely) or make the forwarding {app-id}/{port-index}?